docs/niswitch.rst

```diff
@@ -0,0 +1,18 @@
+.. NI Modular Instruments Python API documentation master file, created by
```

Comment:
This file is the same as the other drivers. Opened https://github.com/ni/nimi-python/issues/258

Reply:
Sounds good

Reply:
They are similar, but they don't change like other rst files based on metadata. For these, you create it once and it will not change.

Also, not all are the same. I.e. modinst doesn't have enums.

Reply:
Ok, we can decide on this separately.